src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/clientlibs/theme/components/navigation/navigation-component.css

```diff
@@ -517,6 +517,30 @@
     animation-name: navigation__slideoutSide;
 }
 
```

Comment:
We have navigation-aem.css for AEM specific custom CSS.
Wouldn't that be a more appropriate location for this CSS?